### PR TITLE
Remove uninformative sentence

### PR DESCRIPTION
**Pull request by remonator**
See commit 7a5a6ba

### CODE REVIEW

**Comment by sypets** [comment]
You cannot make changes to the ViewHelper reference here. The ViewHelper reference is automatically generated from the source code. 

Please see the file CONTRIBUTING.rst in this manual: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-ViewHelper/blob/master/CONTRIBUTING.rst

Contribution is generally very welcome and it's great that you want to contribute. You can contribute to most other TYPO3 manuals. (You will usually see an "Edit on GitHub" button on the top if this is possible), e.g. https://docs.typo3.org/m/typo3/reference-coreapi/10.4/en-us/ExtensionArchitecture/Index.html

See also: https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingDocsOfficial/Index.html

**Comment by remonator** [comment]
Thank you for your answer @sypets!

I'm aware that I have to look into the source code when it comes to the documentation of ViewHelpers - this issue, however, concerns the index page of the ViewHelper documentation section, and after conducting full-text searches in the source code files, this was the only place where I could find the typo mentioned in the PR (in `Index.rst`).

Please let me know if I'm still in the wrong place here...

**Comment by sypets** [comment]
Ah, sorry. My mistake. I did not look closely at your change. I will reopen.

File: Documentation/Index.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ provide a fully useful reference of all provided ViewHelpers by TYPO3 CMS.
 
 Right now this might look ugly in some places, e.g. ``= EXAMPLE =`` and rendered
 HTML markup like ``<code>``. This will be polished in the future.  The current
-state still is much better then nothing.
+state still is much better than nothing.
```
Comment (sypets) [inline Documentation/Index.rst:23]: I would remove the sentence entirely. It does not give any information. Also, before, we had manually created ViewHelper reference, so it was not "nothing".

Comment (remonator) [reply Documentation/Index.rst:23]: You're right. I will adjust the pull request.

**Pull request merged**